components/admin_console/system_users/list/index.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
```

Comment:
This is not entirely relevant but do we have a format for this?

Reply:
Right, I keep tripping over this because I always ignore these comments when editing. Personally, I think they are superfluous, but I'll update to 2018 for the files I've touched.